src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] output = readFile(fromFileName);
+        String result = getResult(output);
+        writeToFile(result, toFileName);
+    }
+
+    private int[] readFile(String sourceFile) {
+        File file = new File(sourceFile);
+        int supplyResult = 0;
+        int buyResult = 0;
+        int result = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String value = reader.readLine();
+            while (value != null) {
+                if (value.contains(SUPPLY)) {
+                    supplyResult += Integer.parseInt(value.substring(value.indexOf(COMMA) + 1));
```

Comment:
it's bad practice to use several expressions in one place. Think about creating separate variable for (value.substring(value.indexOf(COMMA) + 1)